### ISSUE TITLE
pollTimes does not show in UI

### ISSUE THREAD

**Issue by pcxnet** [Joffcom/n8n-nodes-rss-feed-trigger#8]
Can not start or activate workflow since pollTimes does not exist and isn't set.

Error is: 

There was a problem activating the workflow: "Could not get parameter "pollTimes"!"

**Comment by Joffcom** [comment]
Hey @pcxnet 

I am going to close this one as it is not an issue with the node itself and is an n8n issue.

The fix for this is already in and will be available in the next release for now though to use the node 0.226.x should work.

**Comment by Joffcom** [comment]
@pcxnet quick update... 0.228.2 of n8n brings this back.

![image](https://github.com/Joffcom/n8n-nodes-rss-feed-trigger/assets/4688521/f2c87d3f-cb77-42a9-9d84-d9ec33507f2f)

**Comment by pcxnet** [comment]
thank you 👌

**Comment by pcxnet** [comment]
Sorry, im still getting the same thing on 228.2 ?  

I tried deleting the node and putting it back in but same thing . the UI isn't showing and the same error? 
![image](https://github.com/Joffcom/n8n-nodes-rss-feed-trigger/assets/16738529/71daf548-0b99-48b1-88a7-dde5e06cdffa)

**Comment by Joffcom** [comment]
Hey @pcxnet 

That is odd I updated my instance and it was working, I will do some more testing internally to see what is going on.

Are you running n8n in a container or from npm?

**Comment by pcxnet** [comment]
In a container via docker, but literally just did a new install because I changed from mysql to POSTGRESQL. Still in docker, got the issue is in both.

**Comment by LostOnTheLine** [comment]
> Sorry, im still getting the same thing on 228.2 ?
> 
> I tried deleting the node and putting it back in but same thing . the UI isn't showing and the same error?


I've got the same thing. I'm running in Docker, Version 0.228.2, Default Database
I've stopped & restarted the container, I've removed & re-added the trigger, no change. 

Strangely mine will work
```
Workflow activated

Your workflow will now regularly check RSS Feed for events and trigger executions for them.
These executions will not show up immediately in the editor, but you can see them in the execution list if you choose to save executions.
```
So I assume there's _**some**_ Poll Time set, I just can't see what it is or change it...

![RSS Feed no Time](https://github.com/Joffcom/n8n-nodes-rss-feed-trigger/assets/60684934/d44a1721-f212-4282-86fe-cfad9aa9ddea)

I tried removing the add-on, restarting, re-adding the add-on, restarting, then adding the trigger... No change, it still lets me activate it but doesn't give me the ability to add poll Time. Why can't we just use a schedule trigger? Is there a reason the poll time needs to be on this instead of a separate step? Personally I like it all together better... But not if I don't have any control over it

**Comment by Joffcom** [comment]
Hey @LostOnTheLine 

The poll time should be added by the n8n core automatically, this was fixed again recently so the latest release might work now.

You can use the schedule node with the normal rss node to achieve something similar this was put together just to try and make things a bit easier but I wasn't expecting so may issues in the core 🙂

**Comment by LostOnTheLine** [comment]
> The poll time should be added by the n8n core automatically, this was fixed again recently so the latest release might work now.

I'm using the latest release 0.228.2. But the problem is there... As I mentioned in my post. 


> You can use the schedule node with the normal rss node to achieve something similar this was put together just to try and make things a bit easier but I wasn't expecting so may issues in the core 🙂

You can't though. The regular RSS Node will return every item, in my case all 50, every time it runs. There's no simple way to make it not do that, I found a couple but they were overly complicated, to an unreasonable level, & couldn't be copied over. The biggest thing this Node does is make it easy to get only new feeds, that's the part that can't easily be done with preexisting nodes

**Comment by Joffcom** [comment]
Hey @LostOnTheLine 

When I mentioned the latest release I didn't mean the latest tag, the actual latest n8n release is 0.230.1 which has the next tag, this should be fixed in that release although it was also down as being released in 228.2 which you are using but I found it wasn't always successful in that release.

Before I made this node the approach was to use a schedule trigger node, then an rss node then an if or more recently a filter node to check the pubDate against the last run date time the rss node will still return the extra items but the if / filter will let you only pass on the new items this is basically all the node does so rather than 3 nodes there is only one.

If 230.1 doesn't resolve this I will bring it up internally again to have this looked into more.
